src/containers/SignUp/signUp.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react';
+
+import { Input, Icon, Button } from 'antd';
+import PropTypes from 'prop-types';
+import Header from '../../components/Header';
+import GoogleButton from '../../components/GoogleButton';
+import FacebookButton from '../../components/FacebookButton';
+import schema from '../../utils/signup-validation';
+
+import './signUp.css';
+
+class SignUp extends Component {
+  state = {
+    name: '',
+    email: '',
+    password: '',
+    passwordConfirm: '',
+  };
+
+  handleSubmit = async () => {
```

Comment:
#### Good work Fares.
#### But you forget to handle validation error for every input field.
This is my assumption for handling validation error :
  ```
 const objError = {};
        error.inner.forEach(fielderror => {
          objError[fielderror.path] = fielderror.message;
        });
        return this.setState({ errors: objError });
```
#### and to show message error after each input add this : (conditional rendering)
`{errors.title && <span className="error-field">{errors.title}</span>}`